std/haxe/macro/Context.hx

```diff
@@ -395,6 +395,15 @@ class Context {
 		return load("do_parse", 3)(expr, pos, true);
 	}
 
+	/**
+		Parse file content for newlines, allowing positions to be resolved
+		properly inside that file later on (using `Context.parseInlineString`
+		for example). Works with both real and virtual files.
+	**/
+	public static function loadLexerLines(file:String, content:String):Void {
```

Comment:
I would prefer a different name because nobody knows wtf a lexer is. Plus there isn't really any loading involved.

Something like `registerFileContents` would work I think.

Reply:
Yeah I like it better too.